src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProviderTests.cs

```diff
@@ -157,5 +157,75 @@ class Bar : I1
             await VerifyItemExistsAsync(markup, "Prop");
             await VerifyItemExistsAsync(markup, "TestEvent");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task NotInaccessibleMember_01()
```

Comment:
💭 To clarify, this is intentionally `NotInaccessible` (as opposed to `NotAnAccessible`)?

Reply:
The tests (after implementing the feature) should verify that Inaccessible members are Not offered for implementation.

---
In reply to: [270049475](https://github.com/dotnet/roslyn/pull/34529#discussion_r270049475) [](ancestors = 270049475)